src/vivarium_public_health/metrics/disease.py

```diff
@@ -57,7 +50,7 @@ class DiseaseObserver(Component):
     def configuration_defaults(self) -> Dict[str, Any]:
         return {
             "stratification": {
-                self.disease: self.CONFIGURATION_DEFAULTS["stratification"]["disease"]
+                self.disease: super().configuration_defaults["stratification"]["disease"]
```

Comment:
Why do we need this? Isn't just using the superclass `StratifiedObserver` sufficient?

Reply:
No. The super() would return
```
"stratification": {
  "disease": {
    "exclude": [],
    "include": [],
  },
}
```

But we don't want a generic "disease" for the second key, but the specific disease name